template/build/core/files/deploy-exclude.txt

```diff
@@ -0,0 +1,25 @@
+# These files and their children will not be copied to the deployment artifact.
+# If these files already exist in the upstream artifact, they will be removed.
+.drush-use
+.git
+.gitignore
+.travis.yml
+.vagrant
+blt.sh
+bower_components
```

Comment:
Sadly I don't think applies to subdirs, and I haven't had much luck using patterns in the exclude file.